apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -289,13 +315,28 @@ class PageTemplateModal extends Component {
 			return null;
 		}
 
+		// The raw `templates` prop is not filtered to remove Templates that
```

Comment:
@marekhrabe I wonder can you think of a better way to filter the `this.props.templates`? Remember this is the raw blocks before they are parsed and therefore it doesn't have `core/missing`. This is why I'm relying on the `getBlocksByTemplateSlugs` which has been filtered to then filter the `this.props.templates`. Feels a tad brittle though.

Reply:
I've been thinking about this and for the sake of shipping this fix, I would not further complicate this PR with some sort of refactor. I have some ideas in my mind and I will document them in an issue that we might or might not pick up in future.

tldr: let's keep this approach for now, nothing really wrong with it

Reply:
I agreed. Also, have ideas. Please do create that follow up refactor Issue if you can.